src/node/scan-dirs.ts

```
@@ -11,64 +11,74 @@ import pm from 'picomatch'
+  const names: string[] = []
+  let angleDepth = 0
+  let parenDepth = 0
+  let braceDepth = 0
+  let bracketDepth = 0
+  let current = ''
+  let inValue = false
+
+  for (let i = 0; i < rest.length; i++) {
+    const ch = rest[i]
+
+    if (ch === '<') angleDepth++
```

ESLint

[error] Expect newline after if

(antfu/if-newline)